kk/views/base.py

```diff
@@ -33,3 +36,24 @@ def get_url(self, obj):
             url = request.build_absolute_uri(url)
 
         return url
+
+
+class BaseVoteViewSet(object):
```

Comment:
Both comment types are voteable though, so I think this code could just as well live in `BaseCommentViewSet`?